site/src/Librecores/ProjectRepoBundle/Validator/Constraints/UserOrgNameValidator.php

```diff
@@ -76,7 +76,7 @@ class UserOrgNameValidator extends ConstraintValidator
      *
      * @var bool
      */
-    const CHECK_UNIQUE_ORG  = true;
+    const CHECK_UNIQUE_ORG  = false;
```

Comment:
> The unique username check results to the generation of two constraint validation warnings on the user's form and that's why it was originally removed from here.

Yeah, you'll get two validation warnings if you create a user with the same name as an existing user. However, you'll get only one validation error if you create an user with the same name as an organization. However I'd consider that a minor annoyance, compared to us creating an org and a user with the same name :-)

I don't know if there's a better way of having a UNIQUE constraint across two columns on different tables; I didn't find one in Doctrine when I wrote that one. 

We should check if we can safely remove the unique validation constraint from User and Organization, while keeping the DB constraint (which is useful for index generation).



> The unique org name check was removed because keeping it makes it trigger everytime you edit the organization settings and just keep / submit the same name (99.9% of cases). I've tried removing the field from the form as to disallow the editing of the name by users altogether, like we do on projects, but it keeps triggering.

Yeah, validation groups should be the answer here. I don't know it off-head, so if you cannot figure it out let me know and I'll have a look.


> Maybe leave it as is now and make this in a future change-set that is aimed at having more complete constraints?

If we don't figure this out now we could run into users creating orgs which shadow existing users, and the other way around. I think this is something we don't want to happen and cannot really fix in the future without making people unhappy, so let's figure this out before it goes live.